include/filters.php

```diff
@@ -58,7 +60,11 @@ public function __construct( &$polylang ) {
 		add_filter( 'comments_clauses', array( $this, 'comments_clauses' ), 10, 2 );
 
 		// Filters the get_pages function according to the current language
-		add_filter( 'get_pages', array( $this, 'get_pages' ), 10, 2 );
+		if ( ! version_compare( $wp_version, '6.3.alpha', '>=' ) ) {
```

Comment:
No need for the negation :)
We should also add a comment about backward compatibility, to ease the removal of the code later.

```suggestion
		// Backward compatibility with WP < 6.3.
		if ( version_compare( $wp_version, '6.3.alpha', '<' ) ) {
```

Reply:
Isn't the comment just below?

Reply:
![I should buy a boat (and better glasses)](https://github.com/polylang/polylang/assets/1878479/283d5e87-f1c1-42d9-824a-bf81705d9ed1)